cdap-ui/app/directives/widget-container/widget-schema-editor/widget-schema-editor.html

```diff
@@ -1,47 +1,50 @@
-<div class="text-danger" ng-if="error">
-  {{ error }}
-</div>
-
-<div ng-if="properties.length > 0">
-  <div class="clearfix">
-    <div class="col-xs-4">
-      <strong>Name</strong>
-    </div>
-    <div class="col-xs-4">
-      <strong>Type</strong>
-    </div>
-    <div class="col-xs-2 text-center">
-      <strong>Nullable</strong>
-    </div>
+<div class="schema-container">
+  <div class="text-danger" ng-if="error">
+    {{ error }}
   </div>
-  <div class="clearfix" ng-repeat="property in properties">
 
-    <div class="col-xs-4">
-      <input type="text" class="form-control" ng-model="property.name" placeholder="name"/>
+  <div ng-if="properties.length > 0">
+    <div class="clearfix field-labels">
+      <div class="col-xs-5">
+        <strong>Name</strong>
+      </div>
+      <div class="col-xs-5">
+        <strong>Type</strong>
+      </div>
+      <div class="col-xs-1 text-center">
+        <strong>Nullable</strong>
+      </div>
     </div>
+    <div class="clearfix property-row" ng-repeat="property in properties">
```

Comment:
Here too. I thought we pull things to left and right and add a clearfix to the parent to clear both so that subsequent elements are not pulled out. Am I missing something?